tests/plugins/easyimage/commands.js

```diff
@@ -135,6 +135,39 @@
 
 					assert.areSame( 'side', widget.data.type, 'Widget has correct type data' );
 				} );
+			},
+
+			'test balloontoolbar integration': function( editor, bot ) {
+				bot.setData( widgetHtml, function() {
+					var widget = editor.widgets.getByElement( editor.editable().findOne( 'figure' ) ),
+						toolbar = editor.balloonToolbars._contexts[ 0 ].toolbar;
+
+					toolbar._view.once( 'show', function() {
+						resume( function() {
```

Comment:
I find such approach more readable to explicitly wrap every step with `wait`/`resume`. But yes, it could be safely omitted. I've removed redundant `wait`/`resume` pair.